loganalyzer.py

```diff
@@ -440,7 +443,7 @@ def checkWindowsVer(lines):
 
     # This is such a hack, but it's unclear how to do this better
     if verinfo["version"] == "10.0" and "release" not in verinfo:
-        msg = "You are running an unknown Windows 10 release (build %d), which means you are probably using an 'insider' build. Becaue insider builds are test versions, you may have problems that would not happen with release versions of windows." % (
+        msg = "You are running an unknown Windows 10 release (build %d), which means you are probably using an 'insider' build. Some checks that are applicable only to specific Windows versions will not be performed. Also, becaue insider builds are test versions, you may have problems that would not happen with release versions of windows." % (
```

Comment:
> becaue
Typo ;)

Also, change 'insider' to just Insider. And capitalize the final instance of Windows in this message.

Reply:
sigh. There was a point at which I could make software changes that *didn't* have stupid mistakes every single time. Those days appear to be long gone.